examples/shared-vpc/README.md

```diff
@@ -0,0 +1,61 @@
+# Shared VPC Example
+
+This example shows how to use this module in a Shared VPC setup. In a service project, it creates a global HTTP forwarding rule to forward traffic to an instance group. Meanwhile it ensures the host project has firewall rules in place to allow the health check ingress.
+
+## Set up the environment
+
+```
+gcloud auth application-default login
+export GOOGLE_PROJECT=$(gcloud config get-value project)
+```
+
+## Run Terraform
+
+```
+terraform init
+terraform plan
+terraform apply
+```
+
+Open URL of load balancer in browser:
+
+```
+EXTERNAL_IP=$(terraform output -module gce-lb-http external_ip)
+(until curl -sf -o /dev/null http://${EXTERNAL_IP}; do echo "Waiting for Load Balancer... "; sleep 5 ; done) && open http://${EXTERNAL_IP}
+```
+
+> Wait for all instance to become healthy per output of: `gcloud compute backend-services get-health group-http-lb-backend-0 --global`
+
+You should see the instance details from the region closest to you.
+
+### Test balancing to other region
```

Comment:
This example only has 1 managed instance group but there are several references to the second one (copy-paste from another example). My suggestion would be to clean up the readme and tf files to remove references to the second instance group.